test/integration/fleetscope/fleetscope_test.go

```diff
@@ -55,6 +55,7 @@ func TestFleetscope(t *testing.T) {
 				"network_project_id":     multitenant.GetStringOutput("network_project_id"),
 				"fleet_project_id":       multitenant.GetStringOutput("fleet_project_id"),
 				"cluster_membership_ids": testutils.GetBptOutputStrSlice(multitenant, "cluster_membership_ids"),
+				"teams":                  testutils.GetBptOutputStrSlice(setup, "teams")[0],
```

Comment:
I suspect this will work, but here is potential general purpose alternative if needed as you aren't actually using the values in Go:

```suggestion
				"teams":                  setup.GetJsonOutput("teams").Value().(map[string]interface{}),
```

Long term my plan is to add an upstream `GetJsonSetupOutput()` so we don't need the separate `setup`, etc.

Reply:
changed. thanks.